maestro-app/src/main/java/bio/overture/maestro/app/infra/adapter/inbound/messaging/song/SongAnalysisSink.java

```diff
@@ -0,0 +1,19 @@
+package bio.overture.maestro.app.infra.adapter.inbound.messaging.song;
```

Comment:
add licence header for this file and all the others

Reply:
I'll do in a different PR didn't want to modify all files reviewers will suffer to know real changes

Reply:
:+1: